js/index.js

```diff
@@ -40,3 +40,70 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let codesnippet = document.getElementById("cta-img");
+codesnippet.setAttribute('src', siteContent["cta"]["img-src"])
+
+let midsnippet = document.getElementById("middle-img");
+midsnippet.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+// Nav
+const anchors = document.querySelectorAll('a');
+anchors[0].textContent = siteContent.nav['nav-item-1'];
+anchors[1].textContent = siteContent.nav['nav-item-2'];
+anchors[2].textContent = siteContent.nav['nav-item-3'];
+anchors[3].textContent = siteContent.nav['nav-item-4'];
+anchors[4].textContent = siteContent.nav['nav-item-5'];
+anchors[5].textContent = siteContent.nav['nav-item-6'];
+
```

Comment:
try 
`anchors.forEach((anchor,i)=> anchor.textContent = siteContent.nav["nav-item-${i+1}"])`

add backticks to `${}`